integration/app/test-demo/src/components/MeetingEndControl.tsx

```diff
@@ -0,0 +1,24 @@
+import { ControlBarButton, Remove, useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import React from 'react';
+import { endMeeting } from '../utils/api';
+
+const MeetingEndControl = () => {
+  const meetingManager = useMeetingManager();
+
+  const endMeetingForAll = async () => {
+    const meetingId = window.localStorage.getItem('meetingId');
+
+    try {
+      if (meetingId) {
+        await endMeeting(meetingId);
```

Comment:
Should we leave the meeting first before ending it?

Reply:
Also do we need to test this meetingEnd workflow at all in React since it is already covered in JS SDK and there is nothing specific to React with meeting end vs leave.

Reply:
Updated: first leave then end.